test/suites/add.sh

```diff
@@ -91,6 +91,7 @@ test_add_interactive() {
   for m in micro01 micro02 micro03 ; do
     lxc exec "${m}" -- snap disable microovn || true
     lxc exec "${m}" -- snap disable microceph || true
+    lxc exec "${m}" -- snap restart microcloud
```

Comment:
> MicroCloud has an internal loop which tries to check every second which services are present.
When enabling disabled services during the tests, ensure that MicroCloud picks up the enabled services by restarting it.
>
>This might cause issues from time to time on faster runner machines.

I'm not sure I follow this one. 

Firstly, why does Microcloud check every second which services are available, that seems wasteful. Can we figure this out via snap interfaces? 

Second, if it truly is necessary to poll every second, can we instead just sleep for 1s after this loop is complete - at which point we would expect Microcloud to have picked up available services? This would at least test that the polling works and might even be quicker that restarting the snap.